README.md

```diff
@@ -12,6 +12,7 @@
 ### Prerequisites
 
 1. [Python [2.7;3.4]](https://www.python.org/downloads/) installed in your operative system.
+
```

Comment:
@kathyguerron there is no need to add empty spaces in an enumeration. I'll fix this one.